src/chainparams.cpp

```diff
@@ -27,6 +27,16 @@ void ReadSigNetArgs(const ArgsManager& args, CChainParams::SigNetOptions& option
         }
         options.challenge.emplace(ParseHex(signet_challenge[0]));
     }
+    if (args.IsArgSet("-signetblocktime")) {
```

Comment:
Maybe require that `signetblocktime > 0` ? Probably bad things would happen otherwise.

Reply:
Did `signetblocktime >= 0` just to be sure